### PR TITLE
Feature/1788/Add_coming_soon_template_to_leads

### PR DESCRIPTION
**Pull request by Sachinbisht27**
Targets #1788

### Description
I changed lead page theme to ColoredCow main theme and added a coming soon template to it.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by P4NK4J — CHANGES_REQUESTED** [review]
@Sachinbisht27 I don't think you need to create header and everything again. That's already created. You just need to see how you can include them here, probably it would be including any master/app blade file that will bring the changes here. Please correct this and then assign for review.

**Comment by Sachinbisht27** [comment]
@P4NK4J I don't get you. Does i have to make changes in master/app blade file? And What are those major changes if they are not creating headers. Please explain.

**Comment by P4NK4J** [comment]
@Sachinbisht27 If you take a latest pull from master branch, run `npm run dev` and then access the `Task` module from navigation, it will give you some reference, how you can make the leads page similar to it. Let me know if that makes sense

image

File: Modules/AppointmentSlots/Resources/views/layouts/master.blade.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@extends('layouts.app')
+@extends('')
```
Comment (P4NK4J) [inline Modules/AppointmentSlots/Resources/views/layouts/master.blade.php:1]: why this is changed ?

**Review by P4NK4J — CHANGES_REQUESTED** [review]
@Sachinbisht27 Left some comments, also could you share the screenshot of how it is looking on the UI.

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left a comment, please check

**Comment by Sachinbisht27** [comment]
@P4NK4J At first I used HTML code of main coloredcow portal to get the theme for the lead page, then you said me that we don't have to add headers and all again and again then I learned about @extends & @yield used in laravel to get content from other pages and used @extends('layouts.app') in master.blade.php file.
This is my first time on laravel and i did some mistakes.
Now I understand that why we use @extends (which is a laravel tamplate which defines its own sections using @yield) and @yield (used to define section in layout and used to get content from one page to another page).
Attaching new screenshot of master.blade.php file. 
Screenshot (9)

Does I have to make more changes?

**Comment by P4NK4J** [comment]
@Sachinbisht27 No worries, we always learn from our mistakes. Glad you researched a lot about these helpers. If you feel like everything is fine from your end, please push the code and assign it for review. The portal team reviewers would take a look and take action accordingly.

File: Modules/AppointmentSlots/Resources/views/layouts/master.blade.php (hunk 1 above)
Comment (Sachinbisht27) [reply Modules/AppointmentSlots/Resources/views/layouts/master.blade.php:1]: I don't remember when I did these changes in AppointmentSlots.
I don't know sir, I think by mistake.

Comment (Sachinbisht27) [reply Modules/AppointmentSlots/Resources/views/layouts/master.blade.php:1]: I am correcting it and adding new PR

**Review by P4NK4J — COMMENTED** [review]
A minor comment, else looks good

**Comment by Sachinbisht27** [comment]
> @Sachinbisht27 If you take a latest pull from master branch, run `npm run dev` and then access the `Task` module from navigation, it will give you some reference, how you can make the leads page similar to it. Let me know if that makes sense
> 
> image

@P4NK4J  
I searched everywhere and tried so much to align ComingSoon!! text in the middle of the page. But it always aligned in the center of the row not in the center of the page.
Till this morning I haven't did this method which you suggested to do pull req. and all..
Now I saw the [ div class="task-content" ] i did't saw this bootstrap class anywhere.
But now by taking your reference task is done. 

I have a question.
please suggest me any reference where i can learn about <div class="task-content">.

**Comment by Sachinbisht27** [comment]
please suggest me any reference where i can learn about div class="task-content"

**Comment by P4NK4J** [comment]
@Sachinbisht27 `text-center` is a class in Bootstrap used to align the text in the center of the section. In CSS you would right it to
```
div {
text-align: center;
}
```
I told you to replace the `style="margin-top:6cm` with bootstrap class `mt`, if you write it like `mt-2`, it means, that `margin-top: 2*spacer (spacer will be defined in your variables.scss)`.

Let me know if that answers your question.

**Comment by Sachinbisht27** [comment]
> @Sachinbisht27 `text-center` is a class in Bootstrap used to align the text in the center of the section. In CSS you would right it to
> 
> ```
> div {
> text-align: center;
> }
> ```
> 
> I told you to replace the `style="margin-top:6cm` with bootstrap class `mt`, if you write it like `mt-2`, it means, that `margin-top: 2*spacer (spacer will be defined in your variables.scss)`.
> 
> Let me know if that answers your question.

@P4NK4J I tried to replace style with mt already but it allows me to increase by mt-25 only after that the div length ends and the text goes up as it is.

Attaching some SS:

1) When i used mt-25
Screenshot (13)


2) When i used mt-26
Screenshot (14)

**Comment by P4NK4J** [comment]
@Sachinbisht27 mt-26, just see inspect in the browser how much margin is being applied. Why on a different screen is not correct, and how you can use bootstrap media queries to resolve the same. You can refer to Tasks page coming soon template as well.

**Comment by Sachinbisht27** [comment]
@P4NK4J i have added new PR as i take reference from task module from nav.

**Pull request merged**